tests/client_tests/node-postgres/README.rst

```diff
@@ -0,0 +1,107 @@
+====================================
+CrateDB client tests (node-postgres)
+====================================
+
+A nodejs_ application showcasing a test interaction with **CrateDB**.
+
+Staring the node-postgres_ module as middleware.
+
+The interaction:
+
+1. Opens a connection pool with CrateDB.
```

Comment:
I think this is way too much detail / unnecessary information. Also the sub-module / structure description is a bit too much for my taste. This is bound to get out-dated and out of sync. 

If you want to document things, keep it close to artifacts in question. E.g. comments within the file. 

I think for the README we should just have to "How to run it" part. Everything else is overkill

Reply:
ok, I overshot, and I don't say anything that a dev could not figure out by reading the doc.